decidim-core/app/assets/javascripts/decidim/vizzs/areachart.js.es6

```diff
@@ -1,226 +1,204 @@
-/* global d3, DATACHARTS, fetchDatacharts */
+/* global d3 */
 /* eslint-disable id-length, no-unused-vars, multiline-ternary, no-ternary, no-nested-ternary, no-invalid-this */
 /* eslint prefer-reflect: ["error", { "exceptions": ["call"] }] */
 /* eslint dot-location: ["error", "property"] */
 
 // = require d3
 
-const renderAreaCharts = () => {
-  // lib
-  const areachart = (opts = {}) => {
-    // parse opts
-    let data = opts.data
-    let title = opts.title
-    let container = d3.select(opts.container)
-    let showAxis = opts.axis
-    let ratio = opts.ratio
-    let showTooltip = opts.tip !== "false"
-
-    // set the dimensions and margins of the graph
-    let margin = {
-      top: 0,
-      right: 0,
-      bottom: 0,
-      left: 0
-    }
+// lib
+const areachart = (opts = {}) => {
```

Comment:
That exposes areachart in a global scope. It should be wrapped into an object as `exports.Decidim.Visualizations.Areachart`, as OrgChart is done [here](https://github.com/decidim/decidim/blob/7a81c28a4d273736945eea52340ed503fe89a26c/decidim-core/app/assets/javascripts/decidim/vizzs/orgchart.js.es6)

Reply:
I modify this old version of areachart because I need it to be a public constructor, no more reasons involved. Taking care that this is a partial DataVizz implementation, only adding some areacharts to Home Page, we should wait for the rest to add that new implementation.

Reply:
I wouldn't do here neither. As the previous comment, that would be part of a new iteration, thereby new issue, etc...